CHANGELOG.md

```diff
@@ -14,6 +14,14 @@ The format is based on [Keep a Changelog].
 > - **Security**: in case of vulnerabilities.
 
 
+## [UNRELEASED]
+
+### Added
+
+- `IBMQJob.result()` now accepts an optional `refresh` parameter. If 
+  specified, the function re-queries the api for the results, rather 
```

Comment:
nit - can you change `If specified` to `If refresh=true is specified`? Just to be clearer.